fileshare/server.go

```diff
@@ -465,3 +465,18 @@ func (s *Server) SetNotifications(ctx context.Context, in *pb.SetNotificationsRe
 		}
 	}
 }
+
+func (s *Server) PurgeTransfersUntil(ctx context.Context, req *pb.PurgeTransfersUntilRequest) (*pb.Error, error) {
```

Comment:
This doesn't delete the transfers from previous version of fileshare, which are stored in another place. it only deletes the ones from libdrop db. Maybe it should also go over the other list and delete them too.